.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish Package
+
+on:
+  release:
+    types: [published]
```

Comment:
**question:** could you explain what is the release process in this case? The README file seems to be the best place for that.
What I understand is that currently I would have to update the package.json file with the new version, commit and push the change. This can be done by running `npm version` (which also create the tag).
Then I manually create a GitHub release, fill it and this workflow is run to publish to npm.

Is it what you have in mind?
This could be automated as well, but I am OK to deal with the manual procedure as we are not releasing the package very frequently.

Reply:
Auto-generated releases look good,
I will try to modify the workflow, when creating a tag, automatically generate the corresponding release.
The revised publishing process is roughly, reduced steps to manually create release from tag.

1. manually execute `npm version --tag-version-prefix "v" <patch,minor,...>` (Increment the version number and automatically create version tags)
2. `git push origin master --tags`  (publish workflows are triggered by new tags)

There must be an explicit action to release the version to avoid unexpected releases, and now the action is version tag.

Note that the previous version tag does not have a "v" prefix, but I think it's a good idea to add, this can be configured via npmrc.

I'll create a new commit to implementing this feature and updating the README.

Reply:
Nice, I love that!
Thanks